src/components/PostEditor/PostEditor.js

```diff
@@ -147,7 +147,8 @@ export default class PostEditor extends React.Component {
 
   componentDidUpdate (prevProps) {
     const linkPreview = this.props.linkPreview
-    if (get('post.id', this.props) !== get('post.id', prevProps)) {
+    if (get('post.id', this.props) !== get('post.id', prevProps) ||
+        get('post.details', this.props) !== get('post.details', prevProps)) {
```

Comment:
Makes sense to me, yet this can be a touchy area given that if the details are changing while the component is mounted this would  possibly overwrite an in-progress edit. What came-up?

Reply:
hmm, theres a weird thing where if you load a request directly it breaks because we load the openOffersAndRequests for the group which loads some fields for the post but not all the details for that post. then it wasnt updating the post info when the full post got loaded